### ISSUE TITLE
Accessibility: Insufficient contrast in left menu for active item

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#2076]
In the left menu, the active admin section has a slightly darker background than the inactive items - but this is insufficient contrast to convey the information according to WCAG guidelines.

We should add a `border-left` to the active item.

The design team have put together this mockup. Use the left designs, not the right ones (i.e. just adding the border, not new pading)

<img width="1460" height="2148" alt="Image" src="https://github.com/user-attachments/assets/b634dc2b-1e89-4b4b-aaf5-c92f67a9e73a" />

## Acceptance criteria

- [x] New `border-left` added to the active item in the left menu
- [x] border is 3px
- [x] border colour is the same as the icon colour
- [ ] Active item's text has `font-weight` set to `500`

## PRs

- https://github.com/silverstripe/silverstripe-admin/pull/2082
